### PR TITLE
Preparing kbd/version-2.4.0/

### PR DESCRIPTION
**Pull request by ParianGol**

### CODE REVIEW

**Review by OliverFendt — CHANGES_REQUESTED** [review]
@ParianGol Excellent work.
Given the comment in the dbus curation I think the "warranty-disclaimer" conclusions ar based on your lawyers view.
Please resolve my comments, really very good work

File: analysed-packages/kbd/version-2.4.0/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+## Reviewers
+
+The information was reviewed by:
+
+* Mario Schuknecht
+* Oliver Fendt
+
+## Comment
+
+This package is dual licensed under MIT OR GPL-3.0-or-later. To facilitate automatic use, a default license choice of MIT is denoted in the Acknowledgment section of the OSS disclosure document. This shall not restrict the freedom of users to choose either MIT OR GPL-3.0-or-later. For convenience all license texts are provided.
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/kbd/version-2.4.0/README.md]: https://github.com/Open-Source-Compliance/package-analysis/pull/117/commits/452d74cefb901cdb8f2d02a42c4e0d975b72914c
Done, thank you!

File: analysed-packages/kbd/version-2.4.0/kbd-2.4.0-SPDX2TV.spdx
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/kbd/version-2.4.0/kbd-2.4.0-SPDX2TV.spdx:1]: https://github.com/Open-Source-Compliance/package-analysis/pull/117/commits/aa6c58db9aa89c9bb7a032f869e9e1aa07a9dca2

File: analysed-packages/kbd/version-2.4.0/kbd-2.4.0.spdx.json
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/kbd/version-2.4.0/kbd-2.4.0.spdx.json:1]: https://github.com/Open-Source-Compliance/package-analysis/pull/117/commits/aa6c58db9aa89c9bb7a032f869e9e1aa07a9dca2

File: analysed-packages/kbd/version-2.4.0/kbd-2.4.0.spdx.rdf.xml
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/kbd/version-2.4.0/kbd-2.4.0.spdx.rdf.xml:1]: https://github.com/Open-Source-Compliance/package-analysis/pull/117/commits/aa6c58db9aa89c9bb7a032f869e9e1aa07a9dca2

File: analysed-packages/kbd/version-2.4.0/kbd-2.4.0.spdx.yaml
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/kbd/version-2.4.0/kbd-2.4.0.spdx.yaml:1]: https://github.com/Open-Source-Compliance/package-analysis/pull/117/commits/aa6c58db9aa89c9bb7a032f869e9e1aa07a9dca2

File: analysed-packages/kbd/version-2.4.0/kbd-2.4.0-OSS-disclosure.txt [hunk 2]
```diff
@@ -0,0 +1,5136 @@
...
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY, to the extent permitted by law; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+--------------------------------------------------------------------------------
+
+FSFULLR
+
+Copyright 1996-2006 Free Software Foundation, Inc.
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/kbd/version-2.4.0/kbd-2.4.0-OSS-disclosure.txt:2721]: https://github.com/Open-Source-Compliance/package-analysis/pull/117/commits/aa6c58db9aa89c9bb7a032f869e9e1aa07a9dca2

File: analysed-packages/kbd/version-2.4.0/kbd-2.4.0-SPDX2TV.spdx [hunk 3]
```diff
@@ -0,0 +1,16655 @@
...
+LicenseInfoInFile: NOASSERTION
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: kbd-2.4.0.tar.xz/kbd-2.4.0/data/keymaps/i386/qwerty/ua-utf-ws.map
+SPDXID: SPDXRef-item517187
+FileChecksum: SHA1: d13c73f3e2b785854a5a19159c427e78e91119a1
+FileChecksum: SHA256: 49d36c25d99d28980e970d83c9c51be26e7c28045b928d913a888edf43cf00dd
+FileChecksum: MD5: 25901bcde7fef71e773a7757757456
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/kbd/version-2.4.0/kbd-2.4.0-SPDX2TV.spdx]: https://github.com/Open-Source-Compliance/package-analysis/pull/117/commits/aa6c58db9aa89c9bb7a032f869e9e1aa07a9dca2

**Review by OliverFendt — APPROVED** [review]
Excellent @ParianGol , thank you very much

**Comment by ParianGol** [comment]
> @ParianGol Excellent work. Given the comment in the dbus curation I think the "warranty-disclaimer" conclusions ar based on your lawyers view. Please resolve my comments, really very good work

@OliverFendt Thank you very much! We will discuss with our lawyer to provide you a statement also for warranty-disclaimer.

**Pull request merged**